xsuite/src/cli/cmd.test.ts

```diff
@@ -94,6 +96,18 @@ test("new-wallet --wallet wallet.json --password 1234", async () => {
   ]);
 });
 
+test("new-wallet --wallet wallet.json --password 1234 --from-pem wallet.pem", async () => {
+  const walletPath = path.resolve("wallet.json");
+  await run(
+    `new-wallet --wallet ${walletPath} --password 1234 --from-pem ${pemPath}`,
+  );
+  const keystore = Keystore.fromFile_unsafe(walletPath, "1234");
+  const secretKey = UserSecretKey.fromPem(
+    fs.readFileSync(pemPath, { encoding: "utf8" }),
```

Comment:
`fs.readFileSync(pemPath, "utf8")` would work. Same for the readFileSync in the other file

Reply:
Maybe make the `secretKey` contain the output of `.hex()`? Depends on the decision below on the return type of `getSecretKey()`.

Reply:
https://github.com/arda-org/xSuite/pull/70#discussion_r1438924991

👍

Reply:
We are now returning `.hex()`. See [here](https://github.com/arda-org/xSuite/pull/70#discussion_r1439497571) for more details.